src/main/java/org/launchcode/walkabout/models/User.java

```diff
@@ -13,20 +13,20 @@ public class User extends AbstractEntity {
     @NotNull
     private String pwHash;
 
-    public User() {}
+    private static final BCryptPasswordEncoder encoder = new BCryptPasswordEncoder();
```

Comment:
I believe this needs to stay an the bottom since it is an instance method.

Reply:
From what I understand, all the constructors need to be together and above the methods for best practice.